docs/enroute/Melbourne Centre/HUO.md

```diff
@@ -26,14 +26,13 @@ When **HB ADC** or **LT ADC** is offline, the relevant CTR (Class D `SFC` to `A0
 If not providing a top-down service, due to the low level of CTA at these aerodromes, it is best practice to give airways clearance to aircraft at the holding point, to ensure departing aircraft can have uninterrupted climb.
 
 !!! example
-    **JST718:** "Melbourne Centre, JST718, A320, IFR, Taxiing YMHB for YSSY, Runway 12"
+    **JST718:** "Melbourne Centre, JST718, A320, IFR, Taxiing YMHB for YSSY, Runway 12"  
     **HUO:** "JST718, Melbourne Centre, Squawk 3601, Expect clearance via the LATUM2 departure Runway 12, No Reported IFR Traffic, Call me Ready at the Holding Point for Airways Clearance"  
     **JST718:** "Squawk 3601, Wilco, JST718"  
-
+    ...  
```

Comment:
I removed all the "..." between these, if you wanted to make it standard then go ahead. But make sure all the examples are like this

Reply:
It needs to be there otherwise the formatting goes fucky. Also I think it's good to show that time has elapsed between the calls

Reply:
Check the Sydney TCU page and the examples on there where it has elapsed time

Reply:
the formatting is broken on the SY TCU page as well. once you have a line break, it no longer recgonises the ** as bold (idk why)

Reply:
In VS it shows it as if it breaks but on the actual website its not broke

Reply:
i see